packages/callout/src/Callout.tsx

```diff
@@ -21,33 +19,33 @@ export const Variant = {
 export type Variant = typeof Variant[keyof typeof Variant];
 
 const baseStyle = css`
-  background-color: ${uiColors.white};
-  border-radius: 6px;
-  box-shadow: 0px 2px 5px 0 ${transparentize(0.9, uiColors.gray.dark3)};
-  overflow: hidden;
+  font-family: Euclid Circular A, ‘Helvetica Neue’, Helvetica, Arial, sans-serif; // TODO: Refresh – remove when fonts are updated
+  background-color: ${palette.white};
+  border-radius: 12px;
   position: relative;
 
   &:after {
     content: '';
     position: absolute;
-    width: 4px;
-    left: 0px;
+    width: 13px;
+    left: -1px;
     top: -1px;
     bottom: -1px;
-    border-radius: 6px 0px 0px 6px;
+    border-radius: 12px 0px 0px 12px;
   }
 `;
 
 const headerStyle = css`
-  font-family: Akzidenz, ‘Helvetica Neue’, Helvetica, Arial, sans-serif;
   font-size: 12px;
-  font-weight: bold;
-  letter-spacing: 0.8px;
+  font-weight: 700;
+  letter-spacing: 1px;
   line-height: 16px;
-  padding: 12px 24px 12px 52px;
+  padding: 12px 24px 12px 54px;
```

Comment:
Why 54px? I'm seeing 56px in the mocks (unless the extra 2 is somewhere else)

Reply:
actually i'm reverting this back to `52px` since the border width should not be included.